tools/devtools/__init__.py

```
@@ -96,5 +96,21 @@ async def set_settings(request: Request):
     except Exception as e:
         return web.Response(status=500, text=f"Error: {str(e)}")
 
+@server.PromptServer.instance.routes.delete("/devtools/view")
+async def delete_file(request: Request):
+    try:
+        filename = request.rel_url.query['filename']
+        type = request.rel_url.query.get('type', 'output')
```

Ruff (0.15.10)

[error] Variable `type` is shadowing a Python builtin

(A001)